CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+##2016/06/22 - Releasing 2.0.0
+
+puppet-lint 2.0.0 is a breaking change. Specifically, the renaming of the line length test was changed from `80chars` to `140chars`. You may need to adjust your configuration and lint checks. For example:
+    # Line length test is 80 chars in puppet-lint 1.1.0
+    PuppetLint.configuration.send('disable_80chars')
+    # Line length test is 140 chars in puppet-lint 2.x
+    PuppetLint.configuration.send('disable_140chars')
```

Comment:
So, IMO:

disable_80chars == Don't care if a line is > 80 chars --> Implies disable_140chars for backward compat
disable_140chars == Don't care if a line is >140 chars
disable_char_check == Don't care about line length at all